Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -673,6 +673,23 @@ public async Task<SandboxedProcessPipExecutionResult> RunAsync(CancellationToken
                     string arguments = m_pip.Arguments.ToString(m_pipDataRenderer);
                     m_timeout = GetEffectiveTimeout(m_pip.Timeout, m_sandboxConfig.DefaultTimeout, m_sandboxConfig.TimeoutMultiplier);
 
+                    // If ChangeAffectedInputListWrittenFilePath is set, write the change affected inputs of the pip to the file.
+                    if (m_pip.ChangeAffectedInputListWrittenFilePath != default)
+                    {
+                        var changeAffectedInputs = m_pip.GetChangeAffectedInputNames(m_pathTable);

```

Comment:
The affected inputs must be passed from pip executor to this sandboxed process pip executor. That information is not available with the process.

Reply:
It calls the function GetChangeAffectedInputNames() which implemented in Process.cs to compute the affected inputs here. And the Process has the needed information.